dataactvalidator/config/sqlrules/c12_cross_file.sql

```diff
@@ -12,4 +12,8 @@ WHERE ap.submission_id = {}
 		WHERE af.submission_id = ap.submission_id
 		    AND af.piid = ap.piid
 		    AND af.parent_award_id IS NOT DISTINCT FROM ap.parent_award_id
+	) AND NOT EXISTS(
+	  SELECT transaction_obligated_amou
+	  FROM award_financial AS af
+	  WHERE transaction_obligated_amou IS NULL
```

Comment:
Originally, I had the `af.transaction_obligated_amou IS NULL` inside the `NOT EXISTS` and it doesn't work. The test for PIID, ParentAwardID from file D1 exists in file C fails. So I have to move that outside.